README.md

```diff
@@ -25,10 +25,12 @@ And, after pods are scheduled, you can see the results of
 
 ![result](./docs/images/result.jpg)
 
-You can configure the scheduler on the simulator through KubeSchedulerConfiguration.
+You can configure the scheduler on the simulator or by using environment variable through KubeSchedulerConfiguration.
```

Comment:
This change is not needed. (because even if users pass configuration via env, the configuration itself is `KubeSchedulerConfiguration`.

Reply:
ah, ok. I misunderstood.